fala/apps/adviser/forms.py

```diff
@@ -64,6 +64,24 @@ def clean(self):
         data = self.cleaned_data
         if not data.get("postcode") and not data.get("name"):
             raise forms.ValidationError(_("Enter a postcode or an organisation name"))
+        else:
+            msg1 = "This service does not cover "
+            msg2 = "Try a postcode, town or city in England or Wales."
+            x = "England or Wales. "
+            if re.search(r"^\s*[Bb]\s*[Tt]\s*[0-9]", data.get("postcode")):
```

Comment:
To make these regular expression a bit more readable you could assign postcode first and replace all the spaces:
postcode = data.get("postcode").replace(" ", "")

You can ignore the casing of letters in regex and instead pass the flags=re.IGNORECASE to the search method

re.search(r"^BT[0-9]", data.get("postcode"), flags=re.IGNORECASE)

Reply:
Good idea, will change.